datafusion/expr/src/expr_fn.rs

```diff
@@ -747,6 +747,7 @@ scalar_expr!(
 scalar_expr!(CurrentDate, current_date, ,"returns current UTC date as a [`DataType::Date32`] value");
 scalar_expr!(Now, now, ,"returns current timestamp in nanoseconds, using the same value for all instances of now() in same statement");
 scalar_expr!(CurrentTime, current_time, , "returns current UTC time as a [`DataType::Time64`] value");
+scalar_expr!(CurrentTimestamp, current_timestamp, ,"returns current timestamp in nanoseconds, using the same value for all instances of current_timestamp() in same statement");
```

Comment:
I recommend we also note this is an alias for `now()` so people don't get confused about when to use `now()` and when to use `current_timestamp`